### PR TITLE
Task/new postgis datamodels

### PR DESCRIPTION
**Pull request by IvanHdzC**
This PR creates the new datamodels to enable postgis to change the database name according to the incoming requests.

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md [hunk 1]
```diff
@@ -2,7 +2,12 @@
 Content:
 
 * [Functionality](#section1)
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md:4]: Not sure if this is the right .md to do the documentation changes.

ngsi_postgis_sink.md is basically a pointer to ngsi_postgresql_sink.md. Maybe the changes should be done in ngsi_postgresql_sink.md? Or they are Postgis-specific and the new funciontaltiy wouldn't work with plain Postgresql sink?

Comment (IvanHdzC) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md:4]: That's correct, this changes only applies to Postgis sink... Right @manucarrace?

Comment (fgalan) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md:4]: Discussed at skype. Same functionality for Postgresql will come in another PR.

NTC

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md [hunk 2]
```diff
@@ -25,9 +30,86 @@ Colum should be provisioned as type [`geometry`](http://postgis.net/workshops/po
 
 [Top](#top)
 
+### <a name="section1.2"></a>Mapping `NGSIEvent`s to PostgreSQL data structures
+PostgreSQL organizes the data in schemas inside a database that contain tables of data rows. Such organization is exploited by `NGSIPostgreisSink` each time a `NGSIEvent` is going to be persisted.
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md:34]: I'd suggest to add a note in this section, just after the title:

> **NOTE:** by the moment this mapping functionality works only for Postgis sink. However, the same will be implemented for the Postgresql sink itself. In that moment probably this documentation will be moved to the Postgresql sink documentation (as that is the common place for documentation which applies to both sinks)

Comment (IvanHdzC) [reply doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md:37]: Fixed in [6507901](https://github.com/telefonicaid/fiware-cygnus/pull/1874/commits/650790136ad3675975c35cb6410d611230e4e2bd)

**Pull request merged**